Sources/Rendering/Core/InteractorObserver/index.d.ts

```diff
@@ -0,0 +1,38 @@
+import vtkRenderWindowInteractor from '../../../Rendering/Core/RenderWindowInteractor';
+import vtkRenderer from '../../../Rendering/Core/Renderer';
+import { vtkObject } from '../../../interfaces';
+import { Vector3 } from '../../../types';
+import { EventHandler } from '../../../interfaces-additions';
```

Comment:
it should be import { EventHandler } from '../../../interfaces' and maybe combined with vtkObject import